### DISCUSSION TITLE
Introductions

### DISCUSSION THREAD

**Discussion by edakturk14** [BuidlGuidl/batch2.buidlguidl.com#1] [Announcements]
Hey everyone!

It's Eda here, I've been part of the BuidlGuidl for almost 1.5 years! I'm a developer and I spend a lot of time [writing](https://mirror.xyz/edatweets.eth) about different topics and concepts in crypto. I enjoy working on developer education and creating content/documentation.

See you in TG!

**Comment by yonathan9669** [comment]
Hello Team,

I am thrilled to join this vibrant community. My journey into the blockchain realm began recently during the Constellation Chainlink Hackathon, marking my first real foray into understanding the profound capabilities of blockchain technology. Before this, my knowledge of blockchain was limited to its foundational principles: security, transparency, distribution, and immutability.

I owe a tremendous debt of gratitude to @atg.eth, @carletex.eth, and @eda.eth. Your contributions and support have opened the door for me to this extraordinary world filled with individuals committed to creating global solutions that are publicly accessible and trustable. This experience has convinced me that contributing to open-source projects and working towards a trustful, inclusive future is where I want to dedicate my efforts.

Embarking on this path feels like a natural progression in my quest for knowledge and impact. While this is my first venture into contributing to the open-source software (OSS) community, it's a step I've been eager to take for quite some time.

To formally introduce myself: My name is Yonathan Morales, a Venezuelan Software Engineer with a passion for tackling challenging problems and pushing my limits. I find joy in sharing knowledge and believe in the power of collective effort to make the world a better place for everyone.

I am here to learn from all of you, absorb as much as I can, and contribute to our shared goals. I look forward to collaborating, sharing insights, and being a proactive member of this community.

Thank you for welcoming me into this journey. I am eager to contribute and grow alongside each of you.

Warm regards,

Yonathan Morales

**Comment by smrm-dev** [comment]
👋 Hi everyone!

I'm Seyed, a Solidity & Python developer exploring the blockchain world. 🚀 Worked on match engines, zapper contracts, ERC-20s, and played around with lending platforms. Also, a Muon app dev using JavaScript.

Excited to join this crew! Let's talk about smart contracts, share ideas, and create cool stuff together. 🤝

Cheers,
Seyed

**Comment by weslygio** [comment]
Hi everyone!

I'm Wesly. Currently an undergraduate student in informatics, exploring and trying out blockchain technology. Still new to the concept of blockchain, hope we can learn more about this together 😄

**Comment by jsr0000** [comment]
Hello everyone!

I'm Josh. I was studying aerospace engineering, but decided to make the switch to the blockchain world as I find it fascinating and would love to be part of the huge impact it will have on the world. After studying it for some time, I have a good understanding of the core concepts but I am yet to collaborate on a project with others, so this is a very exciting opportunity!🚀🚀
